src/store/Account.ts

```diff
@@ -474,9 +474,13 @@ export default {
             commit('accountsInfo', accountsInfo);
 
             // read signer info to get public key
-            const signerModel = accountsInfo.find((w) => w.address.equals(currentSignerAddress));
+            const signerModel = knownAccounts.find((w) => w.address === currentSignerAddress.plain());
```

Comment:
why use knownAccounts here?

Reply:
Because accountsInfo gets updated in storage on account selection. So it might cause **Failure_Core_Zero_Public_Key** or sending Hash_Lock transactions without an Aggregate one following its confirmation while trying to interact with accounts in same profile that might be virgin or accounts that have received some funds already but, still didn't get selected yet to update their accountsInfo data in storage.